{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/templates/base.html

```diff
@@ -1,20 +1,20 @@
 {%- if cookiecutter.add_sass_with_django_compressor.lower() == 'y' %}
 {%- raw %}{% load compress %}{% endraw %}
 {%- endif %}
-{% raw %}{% load staticfiles i18n %}<!DOCTYPE html>
+{% raw %}{% load staticfiles i18n raven %}<!DOCTYPE html>
 {% get_current_language as LANGUAGE_CODE %}
 <html lang="{{ LANGUAGE_CODE }}">
   <head>
     <meta charset="utf-8">
     <link rel="icon" type="image/x-icon" href="{% static 'images/favicon.png' %}">
     <title>{% block title %}{% endraw %}{{ cookiecutter.project_name }}{% raw %}{% endblock title %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="version" content="{{ site_info.RELEASE_VERSION }}">
     <meta name="description" content="{% block meta_description %}{% endblock meta_description %}">
 
-    {# HTML5 shim, for IE6-8 support of HTML5 elements #}
-    <!--[if lt IE 9]>
-      <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    {% if site_info.IS_RAVEN_INSTALLED %}
```

Comment:
same comment as above, I think this should be wrapped in `if cookiecutter.use_sentry_for_error_reporting`